public/templates/demo.html

```diff
@@ -45,9 +43,35 @@
       href="/assets/css/owl-carousel.css"
     />
 
-    <!-- import the webpage's javascript file -->
+    <!-- Import the webpage's javascript file -->
+    <script defer src="/components/demo.js"></script>
+
+    <!-- jQuery -->
     <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-    <script src="/components/patientSignUp.js" defer></script>
+    <script
+      defer
+      src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"
+    ></script>
+    <script
+      defer
+      src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"
+    ></script>
+
+    <script defer src="/assets/js/jquery-2.1.0.min.js"></script>
+
+    <!-- Bootstrap -->
+    <script defer src="/assets/js/popper.js"></script>
+    <script defer src="/assets/js/bootstrap.min.js"></script>
+
+    <!-- Plugins -->
```

Comment:
What are these "plugins" ? What functionality do they provide in this page? Do we need them?

Reply:
Ping? This is something I didn't quiet understand initially (the relationship between these "plugins" and `custom.js`). This is worth a detailed comment if the plugin imports are going to stay in this page.

Please verify whether we need these imports for this page.

Reply:
So the plugins are js files that contain "animation" for the page. In the new pr, I tested all the plugins and only kept the ones that were relevant!

Reply:
Cool beans.

Reply:
Good.

Can you open issues for things we found in this PR:

1. Clean up plugins: Document uses of plugins (e.g. what functionality they provide) and remove any plugins that aren't needed for pages.
2. Standardize jQuery Import: Migrate to latest 3.5.1 version for all pages

These would be good first issues. Come back and post the issue number here (and then resolve this comment).

Reply:
Okay I did! 

The issue is #68